### ISSUE TITLE
AltSource missing

### ISSUE THREAD

**Issue by cekkr** [rileytestut/AltStore#1462]
Hello,
on macOS Sonoma on Apple Silicon, I cannot find the CLI altsource. In the directory I can find only AltServer and altjit. Someone can help me?

Thank you

**Comment by sepbehroozi** [comment]
Came across this issue when trying to figure out why I'm not seeing the `altsource` file.
Same for me. Any answer on this?

**Comment by BrunoC96** [comment]
Same as me, as today in the Content folder is totally missing the Altsource executable, there is another way to download it, or to use it?

**Comment by dapuzz** [comment]
Join the patreon and get the Beta version

**Comment by thedarkknight197** [comment]
> Join the patreon and get the Beta version

No. The project is open-source. Why I need to pay to have access to a open source resources? Is unacceptable!

**Comment by thedarkknight197** [comment]
> Join the patreon and get the Beta version

i have done but it is missing again. I have downloaded from here: https://www.patreon.com/posts/altstore-2-0rc2-111799616?utm_medium=clipboard_copy&utm_source=copyLink&utm_campaign=postshare_fan&utm_content=web_share

**Comment by shanegillio** [comment]
Hi All! Sorry about this - the AltServer version with this feature was released in beta a while back and because we focused on other things, we haven't yet put it in the public version. You can read about it here: https://www.patreon.com/posts/96819971

We'll try and get this into an official update asap :)
